addon/-private/helpers.js

```diff
@@ -151,28 +153,28 @@ export function normalizeText(text) {
 export function every(jqArray, cb) {
   let arr = jqArray.get();
 
-  return Ember.A(arr).every(function(element) {
+  return A(arr).every(function(element) {
     return cb($(element));
   });
 }
 
 export function map(jqArray, cb) {
   let arr = jqArray.get();
 
-  return Ember.A(arr).map(function(element) {
+  return A(arr).map(function(element) {
     return cb($(element));
   });
 }
 
 /**
- * @private
+ * @public
```

Comment:
I think technically it's still a private function cause it's not re-exported to the consumer

Reply:
Oh yeah, I thought some of the other `@public` helpers weren't exported, but I see I was mistaken. I'll switch this one back.

Reply:
A couple of the other helpers that are exported from the file, but not to the consumer, are marked as `@public` like `findClosestValue` and `objectHasProperty` so it seems that the `@public` and `@private` are just referring to whether it's exported from the file or not, not whether it's exported to consumers of the addon.